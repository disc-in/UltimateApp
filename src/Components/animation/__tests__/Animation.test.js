import React from 'react';
import { render } from '@testing-library/react-native';

import animationSquare from '../../../Fixtures/Animation/AnimationSquare';
import Drill from '../Drill';

import Animation from '../Animation';

describe('<Animation />', () => {
  it('renders correctly', async () => {
    const { toJSON } = render(<Animation animation={new Drill(animationSquare)} />);
    expect(toJSON()).toMatchSnapshot();
  });
});
