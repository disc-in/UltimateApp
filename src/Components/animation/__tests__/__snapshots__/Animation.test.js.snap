// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Animation /> renders correctly 1`] = `
<View>
  <View
    onLayout={[Function]}
    style={
      Object {
        "height": 100,
        "width": 100,
      }
    }
  >
    <View
      style={
        Object {
          "height": 100,
          "width": 100,
        }
      }
    >
      <View
        style={
          Array [
            Object {
              "backgroundColor": "#232b2b",
              "borderWidth": 1,
              "position": "absolute",
            },
            Object {
              "height": 85,
              "left": 95,
              "top": 5,
              "width": 0.425,
            },
          ]
        }
      />
      <View
        style={
          Array [
            Object {
              "backgroundColor": "#232b2b",
              "borderWidth": 1,
              "position": "absolute",
            },
            Object {
              "height": 85,
              "left": 5,
              "top": 5,
              "width": 0.425,
            },
          ]
        }
      />
      <View
        style={
          Array [
            Object {
              "backgroundColor": "#232b2b",
              "borderWidth": 1,
              "position": "absolute",
            },
            Object {
              "height": 0.425,
              "left": 5,
              "top": 5,
              "width": 90,
            },
          ]
        }
      />
      <View
        style={
          Array [
            Object {
              "backgroundColor": "#232b2b",
              "borderWidth": 1,
              "position": "absolute",
            },
            Object {
              "height": 0.425,
              "left": 5,
              "top": 39,
              "width": 90,
            },
          ]
        }
      />
      <Text />
    </View>
    <View
      onMoveShouldSetResponder={[Function]}
      onMoveShouldSetResponderCapture={[Function]}
      onResponderEnd={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderReject={[Function]}
      onResponderRelease={[Function]}
      onResponderStart={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      onStartShouldSetResponderCapture={[Function]}
      style={
        Object {
          "left": 0,
          "position": "absolute",
          "top": 0,
          "transform": Array [
            Object {
              "translateX": 28.958333333333332,
            },
            Object {
              "translateY": 28.958333333333332,
            },
          ],
        }
      }
    >
      <View
        style={
          Array [
            Object {
              "alignItems": "center",
              "backgroundColor": "transparent",
              "borderBottomColor": "#feae19",
              "borderColor": "transparent",
              "borderStyle": "solid",
              "justifyContent": "center",
              "textAlign": "center",
              "textAlignVertical": "center",
            },
            Object {
              "borderBottomWidth": 4.166666666666667,
              "borderLeftWidth": 2.0833333333333335,
              "borderRightWidth": 2.0833333333333335,
            },
          ]
        }
      >
        <Text
          style={
            Object {
              "height": 0,
              "width": 0,
            }
          }
        >
          
        </Text>
      </View>
    </View>
    <View
      onMoveShouldSetResponder={[Function]}
      onMoveShouldSetResponderCapture={[Function]}
      onResponderEnd={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderReject={[Function]}
      onResponderRelease={[Function]}
      onResponderStart={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      onStartShouldSetResponderCapture={[Function]}
      style={
        Object {
          "left": 0,
          "position": "absolute",
          "top": 0,
          "transform": Array [
            Object {
              "translateX": 28.958333333333332,
            },
            Object {
              "translateY": 58.958333333333336,
            },
          ],
        }
      }
    >
      <View
        style={
          Array [
            Object {
              "alignItems": "center",
              "backgroundColor": "transparent",
              "borderBottomColor": "#feae19",
              "borderColor": "transparent",
              "borderStyle": "solid",
              "justifyContent": "center",
              "textAlign": "center",
              "textAlignVertical": "center",
            },
            Object {
              "borderBottomWidth": 4.166666666666667,
              "borderLeftWidth": 2.0833333333333335,
              "borderRightWidth": 2.0833333333333335,
            },
          ]
        }
      >
        <Text
          style={
            Object {
              "height": 0,
              "width": 0,
            }
          }
        >
          
        </Text>
      </View>
    </View>
    <View
      onMoveShouldSetResponder={[Function]}
      onMoveShouldSetResponderCapture={[Function]}
      onResponderEnd={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderReject={[Function]}
      onResponderRelease={[Function]}
      onResponderStart={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      onStartShouldSetResponderCapture={[Function]}
      style={
        Object {
          "left": 0,
          "position": "absolute",
          "top": 0,
          "transform": Array [
            Object {
              "translateX": 58.958333333333336,
            },
            Object {
              "translateY": 28.958333333333332,
            },
          ],
        }
      }
    >
      <View
        style={
          Array [
            Object {
              "alignItems": "center",
              "backgroundColor": "transparent",
              "borderBottomColor": "#feae19",
              "borderColor": "transparent",
              "borderStyle": "solid",
              "justifyContent": "center",
              "textAlign": "center",
              "textAlignVertical": "center",
            },
            Object {
              "borderBottomWidth": 4.166666666666667,
              "borderLeftWidth": 2.0833333333333335,
              "borderRightWidth": 2.0833333333333335,
            },
          ]
        }
      >
        <Text
          style={
            Object {
              "height": 0,
              "width": 0,
            }
          }
        >
          
        </Text>
      </View>
    </View>
    <View
      onMoveShouldSetResponder={[Function]}
      onMoveShouldSetResponderCapture={[Function]}
      onResponderEnd={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderReject={[Function]}
      onResponderRelease={[Function]}
      onResponderStart={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      onStartShouldSetResponderCapture={[Function]}
      style={
        Object {
          "left": 0,
          "position": "absolute",
          "top": 0,
          "transform": Array [
            Object {
              "translateX": 58.958333333333336,
            },
            Object {
              "translateY": 58.958333333333336,
            },
          ],
        }
      }
    >
      <View
        style={
          Array [
            Object {
              "alignItems": "center",
              "backgroundColor": "transparent",
              "borderBottomColor": "#feae19",
              "borderColor": "transparent",
              "borderStyle": "solid",
              "justifyContent": "center",
              "textAlign": "center",
              "textAlignVertical": "center",
            },
            Object {
              "borderBottomWidth": 4.166666666666667,
              "borderLeftWidth": 2.0833333333333335,
              "borderRightWidth": 2.0833333333333335,
            },
          ]
        }
      >
        <Text
          style={
            Object {
              "height": 0,
              "width": 0,
            }
          }
        >
          
        </Text>
      </View>
    </View>
    <View
      onMoveShouldSetResponder={[Function]}
      onMoveShouldSetResponderCapture={[Function]}
      onResponderEnd={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderReject={[Function]}
      onResponderRelease={[Function]}
      onResponderStart={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      onStartShouldSetResponderCapture={[Function]}
      style={
        Object {
          "left": 0,
          "position": "absolute",
          "top": 0,
          "transform": Array [
            Object {
              "translateX": 43.958333333333336,
            },
            Object {
              "translateY": 18.958333333333332,
            },
          ],
        }
      }
    >
      <View
        style={
          Array [
            Object {
              "alignItems": "center",
              "backgroundColor": "transparent",
              "borderBottomColor": "#feae19",
              "borderColor": "transparent",
              "borderStyle": "solid",
              "justifyContent": "center",
              "textAlign": "center",
              "textAlignVertical": "center",
            },
            Object {
              "borderBottomWidth": 4.166666666666667,
              "borderLeftWidth": 2.0833333333333335,
              "borderRightWidth": 2.0833333333333335,
            },
          ]
        }
      >
        <Text
          style={
            Object {
              "height": 0,
              "width": 0,
            }
          }
        >
          
        </Text>
      </View>
    </View>
    <View
      onMoveShouldSetResponder={[Function]}
      onMoveShouldSetResponderCapture={[Function]}
      onResponderEnd={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderReject={[Function]}
      onResponderRelease={[Function]}
      onResponderStart={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      onStartShouldSetResponderCapture={[Function]}
      style={
        Object {
          "left": 0,
          "position": "absolute",
          "top": 0,
          "transform": Array [
            Object {
              "translateX": 18.958333333333332,
            },
            Object {
              "translateY": 43.958333333333336,
            },
          ],
        }
      }
    >
      <View
        style={
          Array [
            Object {
              "alignItems": "center",
              "backgroundColor": "transparent",
              "borderBottomColor": "#feae19",
              "borderColor": "transparent",
              "borderStyle": "solid",
              "justifyContent": "center",
              "textAlign": "center",
              "textAlignVertical": "center",
            },
            Object {
              "borderBottomWidth": 4.166666666666667,
              "borderLeftWidth": 2.0833333333333335,
              "borderRightWidth": 2.0833333333333335,
            },
          ]
        }
      >
        <Text
          style={
            Object {
              "height": 0,
              "width": 0,
            }
          }
        >
          
        </Text>
      </View>
    </View>
    <View
      onMoveShouldSetResponder={[Function]}
      onMoveShouldSetResponderCapture={[Function]}
      onResponderEnd={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderReject={[Function]}
      onResponderRelease={[Function]}
      onResponderStart={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      onStartShouldSetResponderCapture={[Function]}
      style={
        Object {
          "left": 0,
          "position": "absolute",
          "top": 0,
          "transform": Array [
            Object {
              "translateX": 43.958333333333336,
            },
            Object {
              "translateY": 68.95833333333333,
            },
          ],
        }
      }
    >
      <View
        style={
          Array [
            Object {
              "alignItems": "center",
              "backgroundColor": "transparent",
              "borderBottomColor": "#feae19",
              "borderColor": "transparent",
              "borderStyle": "solid",
              "justifyContent": "center",
              "textAlign": "center",
              "textAlignVertical": "center",
            },
            Object {
              "borderBottomWidth": 4.166666666666667,
              "borderLeftWidth": 2.0833333333333335,
              "borderRightWidth": 2.0833333333333335,
            },
          ]
        }
      >
        <Text
          style={
            Object {
              "height": 0,
              "width": 0,
            }
          }
        >
          
        </Text>
      </View>
    </View>
    <View
      onMoveShouldSetResponder={[Function]}
      onMoveShouldSetResponderCapture={[Function]}
      onResponderEnd={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderReject={[Function]}
      onResponderRelease={[Function]}
      onResponderStart={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      onStartShouldSetResponderCapture={[Function]}
      style={
        Object {
          "left": 0,
          "position": "absolute",
          "top": 0,
          "transform": Array [
            Object {
              "translateX": 68.95833333333333,
            },
            Object {
              "translateY": 43.958333333333336,
            },
          ],
        }
      }
    >
      <View
        style={
          Array [
            Object {
              "alignItems": "center",
              "backgroundColor": "transparent",
              "borderBottomColor": "#feae19",
              "borderColor": "transparent",
              "borderStyle": "solid",
              "justifyContent": "center",
              "textAlign": "center",
              "textAlignVertical": "center",
            },
            Object {
              "borderBottomWidth": 4.166666666666667,
              "borderLeftWidth": 2.0833333333333335,
              "borderRightWidth": 2.0833333333333335,
            },
          ]
        }
      >
        <Text
          style={
            Object {
              "height": 0,
              "width": 0,
            }
          }
        >
          
        </Text>
      </View>
    </View>
    <View
      onMoveShouldSetResponder={[Function]}
      onMoveShouldSetResponderCapture={[Function]}
      onResponderEnd={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderReject={[Function]}
      onResponderRelease={[Function]}
      onResponderStart={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      onStartShouldSetResponderCapture={[Function]}
      style={
        Object {
          "left": 0,
          "position": "absolute",
          "top": 0,
          "transform": Array [
            Object {
              "translateX": 25.833333333333332,
            },
            Object {
              "translateY": 25.833333333333332,
            },
          ],
        }
      }
    >
      <View
        style={
          Array [
            Object {
              "alignItems": "center",
              "justifyContent": "center",
              "textAlign": "center",
              "textAlignVertical": "center",
            },
            Object {
              "backgroundColor": "#2a9e91",
            },
            Object {
              "borderRadius": 8.333333333333334,
              "height": 8.333333333333334,
              "width": 8.333333333333334,
            },
          ]
        }
      >
        <Text
          style={
            Object {
              "color": "#fff",
              "fontWeight": "bold",
            }
          }
        >
          1
        </Text>
      </View>
    </View>
    <View
      onMoveShouldSetResponder={[Function]}
      onMoveShouldSetResponderCapture={[Function]}
      onResponderEnd={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderReject={[Function]}
      onResponderRelease={[Function]}
      onResponderStart={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      onStartShouldSetResponderCapture={[Function]}
      style={
        Object {
          "left": 0,
          "position": "absolute",
          "top": 0,
          "transform": Array [
            Object {
              "translateX": 11.833333333333332,
            },
            Object {
              "translateY": 25.833333333333332,
            },
          ],
        }
      }
    >
      <View
        style={
          Array [
            Object {
              "alignItems": "center",
              "justifyContent": "center",
              "textAlign": "center",
              "textAlignVertical": "center",
            },
            Object {
              "backgroundColor": "#2a9e91",
            },
            Object {
              "borderRadius": 8.333333333333334,
              "height": 8.333333333333334,
              "width": 8.333333333333334,
            },
          ]
        }
      >
        <Text
          style={
            Object {
              "color": "#fff",
              "fontWeight": "bold",
            }
          }
        >
          2
        </Text>
      </View>
    </View>
    <View
      onMoveShouldSetResponder={[Function]}
      onMoveShouldSetResponderCapture={[Function]}
      onResponderEnd={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderReject={[Function]}
      onResponderRelease={[Function]}
      onResponderStart={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      onStartShouldSetResponderCapture={[Function]}
      style={
        Object {
          "left": 0,
          "position": "absolute",
          "top": 0,
          "transform": Array [
            Object {
              "translateX": 55.833333333333336,
            },
            Object {
              "translateY": 25.833333333333332,
            },
          ],
        }
      }
    >
      <View
        style={
          Array [
            Object {
              "alignItems": "center",
              "justifyContent": "center",
              "textAlign": "center",
              "textAlignVertical": "center",
            },
            Object {
              "backgroundColor": "#2a9e91",
            },
            Object {
              "borderRadius": 8.333333333333334,
              "height": 8.333333333333334,
              "width": 8.333333333333334,
            },
          ]
        }
      >
        <Text
          style={
            Object {
              "color": "#fff",
              "fontWeight": "bold",
            }
          }
        >
          3
        </Text>
      </View>
    </View>
    <View
      onMoveShouldSetResponder={[Function]}
      onMoveShouldSetResponderCapture={[Function]}
      onResponderEnd={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderReject={[Function]}
      onResponderRelease={[Function]}
      onResponderStart={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      onStartShouldSetResponderCapture={[Function]}
      style={
        Object {
          "left": 0,
          "position": "absolute",
          "top": 0,
          "transform": Array [
            Object {
              "translateX": 55.833333333333336,
            },
            Object {
              "translateY": 55.833333333333336,
            },
          ],
        }
      }
    >
      <View
        style={
          Array [
            Object {
              "alignItems": "center",
              "justifyContent": "center",
              "textAlign": "center",
              "textAlignVertical": "center",
            },
            Object {
              "backgroundColor": "#2a9e91",
            },
            Object {
              "borderRadius": 8.333333333333334,
              "height": 8.333333333333334,
              "width": 8.333333333333334,
            },
          ]
        }
      >
        <Text
          style={
            Object {
              "color": "#fff",
              "fontWeight": "bold",
            }
          }
        >
          4
        </Text>
      </View>
    </View>
    <View
      onMoveShouldSetResponder={[Function]}
      onMoveShouldSetResponderCapture={[Function]}
      onResponderEnd={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderReject={[Function]}
      onResponderRelease={[Function]}
      onResponderStart={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      onStartShouldSetResponderCapture={[Function]}
      style={
        Object {
          "left": 0,
          "position": "absolute",
          "top": 0,
          "transform": Array [
            Object {
              "translateX": 25.833333333333332,
            },
            Object {
              "translateY": 55.833333333333336,
            },
          ],
        }
      }
    >
      <View
        style={
          Array [
            Object {
              "alignItems": "center",
              "justifyContent": "center",
              "textAlign": "center",
              "textAlignVertical": "center",
            },
            Object {
              "backgroundColor": "#2a9e91",
            },
            Object {
              "borderRadius": 8.333333333333334,
              "height": 8.333333333333334,
              "width": 8.333333333333334,
            },
          ]
        }
      >
        <Text
          style={
            Object {
              "color": "#fff",
              "fontWeight": "bold",
            }
          }
        >
          5
        </Text>
      </View>
    </View>
    <View
      onMoveShouldSetResponder={[Function]}
      onMoveShouldSetResponderCapture={[Function]}
      onResponderEnd={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderReject={[Function]}
      onResponderRelease={[Function]}
      onResponderStart={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      onStartShouldSetResponderCapture={[Function]}
      style={
        Object {
          "left": 0,
          "position": "absolute",
          "top": 0,
          "transform": Array [
            Object {
              "translateX": 33.22222222222222,
            },
            Object {
              "translateY": 33.22222222222222,
            },
          ],
        }
      }
    >
      <View
        style={
          Array [
            Object {
              "alignItems": "center",
              "backgroundColor": "white",
              "borderColor": "black",
              "justifyContent": "center",
              "textAlign": "center",
              "textAlignVertical": "center",
            },
            Object {
              "borderRadius": 5.555555555555556,
              "borderWidth": 0.5555555555555556,
              "height": 5.555555555555556,
              "width": 5.555555555555556,
            },
          ]
        }
      />
    </View>
  </View>
  <View>
    <View
      style={
        Object {
          "flexDirection": "row",
          "paddingHorizontal": 10,
        }
      }
    >
      <View
        accessible={true}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "marginHorizontal": 10,
            "opacity": 1,
          }
        }
      >
        <Text />
      </View>
      <View
        style={
          Object {
            "flex": 1,
            "marginVertical": 10,
          }
        }
      >
        <View
          style={
            Array [
              Object {
                "backgroundColor": "gray",
                "height": 5,
                "position": "absolute",
                "top": 10,
              },
              Object {
                "width": 12.5,
              },
            ]
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "black",
              "height": 5,
              "position": "absolute",
              "top": 10,
              "width": 0,
            }
          }
        />
        <View
          left={0}
          style={
            Object {
              "backgroundColor": "white",
              "height": 5,
              "position": "absolute",
              "top": 10,
              "width": 5,
            }
          }
        />
        <View
          left={2.5}
          style={
            Object {
              "backgroundColor": "white",
              "height": 5,
              "position": "absolute",
              "top": 10,
              "width": 5,
            }
          }
        />
        <View
          left={5}
          style={
            Object {
              "backgroundColor": "white",
              "height": 5,
              "position": "absolute",
              "top": 10,
              "width": 5,
            }
          }
        />
        <View
          left={7.5}
          style={
            Object {
              "backgroundColor": "white",
              "height": 5,
              "position": "absolute",
              "top": 10,
              "width": 5,
            }
          }
        />
        <View
          left={10}
          style={
            Object {
              "backgroundColor": "white",
              "height": 5,
              "position": "absolute",
              "top": 10,
              "width": 5,
            }
          }
        />
        <View
          left={12.5}
          style={
            Object {
              "backgroundColor": "white",
              "height": 5,
              "position": "absolute",
              "top": 10,
              "width": 5,
            }
          }
        />
        <View
          accessible={true}
          focusable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "height": "100%",
              "left": 0,
              "opacity": 1,
              "position": "absolute",
              "width": 0,
            }
          }
        />
        <View
          accessible={true}
          focusable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "height": "100%",
              "left": 0,
              "opacity": 1,
              "position": "absolute",
              "width": 2.5,
            }
          }
        />
        <View
          accessible={true}
          focusable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "height": "100%",
              "left": 2.5,
              "opacity": 1,
              "position": "absolute",
              "width": 2.5,
            }
          }
        />
        <View
          accessible={true}
          focusable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "height": "100%",
              "left": 5,
              "opacity": 1,
              "position": "absolute",
              "width": 2.5,
            }
          }
        />
        <View
          accessible={true}
          focusable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "height": "100%",
              "left": 7.5,
              "opacity": 1,
              "position": "absolute",
              "width": 2.5,
            }
          }
        />
        <View
          accessible={true}
          focusable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "height": "100%",
              "left": 10,
              "opacity": 1,
              "position": "absolute",
              "width": 2.5,
            }
          }
        />
      </View>
    </View>
  </View>
</View>
`;
